examples/set-bucket-lifecycle.mjs

```diff
@@ -40,12 +40,8 @@ const lifecycleConfig = {
   ],
 }
 
-s3Client.setBucketLifecycle('bucketname', lifecycleConfig, function (err) {
-  if (err) {
-    return console.log(err)
-  }
-  console.log('Success')
-})
+const ilmExpiryDays = await s3Client.setBucketLifecycle('test-bucket', ilmPolicyExpDays)
+console.log(ilmExpiryDays)
```

Comment:
The `console.log` statements in this example seem to be returning `undefined` for me. This is expected, right?